IBAN.md

```diff
@@ -0,0 +1,126 @@
+﻿# International Bank Account Number
+An overview of all IBAN's supported. Note that for countries not mentioned, IBA
```

Comment:
Missing text? The sentence seems to stop abruptly

Reply:
Is this better?

Reply:
Yes